eudemo/eudemo/pf_coils/designer.py

```diff
@@ -121,6 +119,20 @@ def __init__(
         super().__init__(params, build_config)
         self.tf_coil_boundary = tf_coil_boundary
         self.keep_out_zones = keep_out_zones
+        self.file_path = self.build_config.get("file_path", None)
+
+    def read(self) -> CoilSet:
+        """
+        Read in a coilset
+        """
+        if self.file_path is None:
+            raise ValueError("No file path to read from!")
+
+        with open(self.file_path, "r") as file:
+            data = json.load(file)
+
+        eqdsk = EQDSKInterface(**data[next(iter(data))])
```

Comment:
yeah, I ended up putting two equilibria in a single json file to avoid having separate file_folder and file_name. So this is extracting the first json from the file. Not exactly the cutest solution, but nothing better immediately came to mind.